docs/framework/winforms/controls/default-keyboard-and-mouse-handling-in-the-windows-forms-datagridview-control.md

```diff
@@ -53,7 +49,8 @@ The following tables describe how users can interact with the <xref:System.Windo
 |CTRL+HOME|Moves the focus to the first cell in the control.|  
 |CTRL+END|Moves the focus to the last cell in the control.|  
 |CTRL+PAGE DOWN/UP|Same as PAGE DOWN or PAGE UP.|  
-|F2|Puts the current cell into cell edit mode if the <xref:System.Windows.Forms.DataGridView.EditMode%2A> property value is <xref:System.Windows.Forms.DataGridViewEditMode.EditOnF2> or <xref:System.Windows.Forms.DataGridViewEditMode.EditOnKeystrokeOrF2>.|  
+|F2|Puts the current cell into cell edit mode if the <xref:System.Windows.Forms.DataGridView.EditMode%2A> property value is <xref:System.Windows.Forms.DataGridViewEditMode.EditOnF2> or <xref:System.Windows.Forms.DataGridViewEditMode.EditOnKeystrokeOrF2>.|
+|F3|Sorts the current column if the <xref:System.Windows.Forms.DataGridViewColumn.SortMode%2A?displayProperty=nameWithType> property value is <xref:System.Windows.Forms.DataGridViewColumnSortMode.Automatic>. It's the same as clicking the current column header. Available since .NET Framework 4.7.2.|  
```

Comment:
@mairaw - This feature is available only for applications that are re-targeted to 4.7.2 or that opted-in into this behavior using AppContext switches. I'm not sure if we should repeat this info here or add a pointer to a topic about accessibility changes.

Reply:
I haven't found a topic describing how to opt in into 4.7.2 improvements. I think as soon as we have one we should insert it. This is for 4.7.1: https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/retargeting/4.7-4.7.1#accessibility-improvements-in-windows-forms-controls

Reply:
We haven't started the 4.7.2 topics yet @foreverest

Reply:
While we don't have the topic to link to, how about: _Available since .NET Framework 4.7.2 for applications that are built to target 4.7.2 framework or that explicitly opt into accessibility improvements._